applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_isotropic_damage.cpp

```diff
@@ -95,8 +95,8 @@ void GenericSmallStrainIsotropicDamage<TConstLawIntegratorType>::CalculateMateri
         this->CalculateValue(rValues, CONSTITUTIVE_MATRIX, r_constitutive_matrix);
 
         // Converged values
-        double& threshold = this->GetThreshold();
```

Comment:
maybe using const more extensively would help...

Reply:
No, because later the value is changed, but only updated at the end of the loop